src/app/services/auth-service.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  userIsLogged = false;
+  public wrongData: boolean = false;
```

Comment:
variables are public by default, if you prefer approach witch setting accessibility for each variable, do thin everywhere. The same with type. When You are setting value of variable in the same place as declaration you dont need to set type - its set automatically.